test/VoucherHub.test.ts

```diff
@@ -84,9 +86,12 @@ describe('VoucherHub', function () {
                 .to.equal(await voucherHub.defaultAdmin())
                 .to.equal(admin);
             expect(await voucherHub.defaultAdminDelay()).to.equal(0);
-            expect(await voucherHub.hasRole(await voucherHub.UPGRADER_ROLE.staticCall(), admin));
-            expect(await voucherHub.hasRole(await voucherHub.MANAGER_ROLE.staticCall(), manager));
-            expect(await voucherHub.hasRole(await voucherHub.MINTER_ROLE.staticCall(), minter));
+            expect(await voucherHub.hasRole(await voucherHub.UPGRADER_ROLE.staticCall(), admin)).to
+                .be.true;
```

Comment:
expect was no working before this `to.be.true` update

Reply:
why did it work before then ?

Reply:
It was not really checking, it was more like console.log() ..!